README.md

```diff
@@ -105,8 +105,20 @@ kind: BundleInstance
 metadata:
   name: my-bundle-instance
 spec:
-  bundleName: my-bundle
   provisionerClassName: core.rukpak.io/plain
+  selector:
```

Comment:
Is it worth adding a block describing what exactly this selector is selecting? Essentially, I think we should add some documentation around what the embedded `Bundle` is and how a user uses it since it isn't initially intuitive.

Reply:
Ended up dropping this field as it wasn't clear whether we'll need to support it in the short term.